MCSync/frmConfig.cs

```diff
@@ -112,7 +89,16 @@ private void cmdModsWhitelist_Click(object sender, EventArgs e)
             whitelist.Text = "Mods Whitelist";
             whitelist.WhitelistName = "mods_whitelist";
             whitelist.lblIntro.Text = "Edit your Mods whitelist.";
-            whitelist.ShowDialog();
+
+            whitelist.TopLevel = false;
+            whitelist.ControlBox = false;
+            whitelist.Dock = DockStyle.Fill;
+            whitelist.FormBorderStyle = FormBorderStyle.None;
```

Comment:
Should we actually just set these on the form itself rather than doing it here?

Reply:
Fixed